README.md

```diff
@@ -77,6 +77,24 @@ bin/elixirc lib/elixir/lib/string.ex -o lib/elixir/ebin
 bin/elixir lib/elixir/test/elixir/string_test.exs
 ```
 
+To recompile only changed files (including Erlang modules):
```

Comment:
I think we should either say `make erlang` for compiling only Erlang or remove this altogether.